ifrs17/Import/ImportStorage.ipynb

```diff
@@ -341,7 +341,7 @@
                 "\n    public IEnumerable<IfrsVariable> GetIfrsVariables(string dataNode) => IfrsVariablesByImportIdentity.TryGetValue(dataNode, out var variableCollection) ? variableCollection : Enumerable.Empty<IfrsVariable>();",
                 "\n    ",
                 "\n    public double[] GetValues(ImportIdentity id, Func<RawVariable, bool> whereClause) => GetRawVariables(id.DataNode).Where(v => (v.AocType, v.Novelty) == id.AocStep && whereClause(v)).Aggregate()?.Values ?? Enumerable.Empty<double>().ToArray();",
-                "\n    public double GetValue(ImportIdentity id, Func<IfrsVariable, bool> whereClause) => GetIfrsVariables(id.DataNode).Where(v => (v.AocType, v.Novelty) == id.AocStep && whereClause(v)).Aggregate()?.Value ?? 0;",
+                "\n    public double GetValue(ImportIdentity id, Func<IfrsVariable, bool> whereClause) => GetIfrsVariables(id.DataNode).Where(v => (v.AocType, v.Novelty) == id.AocStep && whereClause(v)).Aggregate()?.Value.ElementAtOrDefault(0) ?? 0;",
```

Comment:
There are exceptions to this behavior:
- in the case of GetValue for secondary scope, the value of the projection period in the identity should be taken.

We need to research whether this should be the general behavior.

Reply:
And it likely gives a non-negligible contributions to the failing tests...